src/main/java/com/marek/onlinebookstore/security/token/JwtUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.marek.onlinebookstore.security.token;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Date;
+import java.util.function.Function;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtUtil {
+    @Value("${jwtUtil.expiration}")
+    private long expiration;
+
+    private Key secret;
+
+    public JwtUtil(@Value("${jwtUtil.secret}") String secretString) {
+        secret = Keys.hmacShaKeyFor(secretString.getBytes(StandardCharsets.UTF_8));
+    }
+
+    public String generateToken(String username) {
+        return Jwts.builder()
```

Comment:
You can consider adding here more user information like roles, or other user-specific information.